src/services/notification/index.js

```diff
@@ -13,10 +13,15 @@ const mapResponseProps = (response) => {
 	return response;
 };
 
-const toQueryString = (paramsObject) => Object
-	.keys(paramsObject)
-	.map((key) => `${encodeURIComponent(key)}=${encodeURIComponent(paramsObject[key])}`)
-	.join('&');
+const toQueryString = (paramsObject) =>
+	Object.keys(paramsObject)
+		.map(
+			(key) =>
+				`${encodeURIComponent(key)}=${encodeURIComponent(
```

Comment:
Codacy found an issue: [Expected no linebreak before this expression.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)